app/config/config.yml

```diff
@@ -69,3 +69,59 @@ swiftmailer:
 sensio_framework_extra:
    router:
         annotations: false
+
+easy_admin:
```

Comment:
you should create a new file as 'admin.yml', import that file from this config.yml file and move these easyadmin configuration to admin.yml file.